legend-engine-xt-relationalStore-pure/src/main/resources/core_relational/relational/sqlQueryToString/prestoExtension.pure

```diff
@@ -121,7 +126,7 @@ function <<access.private>> meta::relational::functions::sqlQueryToString::prest
 
 function <<access.private>> meta::relational::functions::sqlQueryToString::presto::schemaNameToIdentifierForPresto(schemaName: String[1], dbConfig: DbConfig[1]): String[1]
 {
-   $schemaName->replace('$', '.');
+   $schemaName->replace('$', '$$$');
```

Comment:
Is the schema name handling change related to quoting of identifiers with spaces ? If not, please submit this as a separate PR.

Reply:
I started off with handling just spaces, but later on came across user cases that required special characters handling as well. I  have changed the name of PR to reflect the same